content/applications/inventory_and_mrp/inventory/barcode/operations/receipts_deliveries.rst

```diff
@@ -0,0 +1,183 @@
+:guilabel:`Product` line under the :guilabel:`Products` tab, click :guilabel:`Add a product`, and
+select the desired product(s) to add to the quotation.
+
+Once ready, click :guilabel:`Save` at the top of the form, and click :guilabel:`Confirm Order` to
+confirm the :abbr:`RFQ (request for quotation)` to a :abbr:`PO (purchase order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-purchase-order.png
+   :align: center
+   :alt: Completed purchase order for barcode product.
+
+To process and scan barcodes for warehouse receipts, navigate to the :menuselection:`Barcode app`.
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To process receipts, click on the :guilabel:`Operations` button at the bottom
+of the screen. This navigates to an :menuselection:`Operations` overview page.
+
+.. image:: receipts_deliveries/receipts-deliveries-barcode-scanner.png
+   :align: center
+   :alt: Barcode app start screen with scanner.
+
+From this page, locate the :guilabel:`Receipts` card, and click the :guilabel:`# To Process` button
+to view all outstanding receipts. Then, select the desired receipt operation to process. This
+navigates to the barcode transfer screen.
+
+.. note::
+   If using only a barcode scanner or the Odoo mobile app, the barcodes for each transfer
+   of a corresponding operation type can be scanned to be processed easily. Once scanned, the
+   products that are part of an existing transfer can be scanned, and new products can be added to
+   the transfer, as well. Once all products have been scanned, validate the transfer to proceed with
+   the stock moves.
+
+From this screen, an overview of all receipts to process within that transfer (**WH/IN/000XX**) is
+shown. At the bottom of the screen, there are options to :guilabel:`Add Product` or
+:guilabel:`Validate`, depending on if products need to be added to the operation, or if the whole
+operation should be validated at once.
+
+.. image:: receipts_deliveries/receipts-deliveries-scanner-overview.png
+   :align: center
+   :alt: Overview of receipts in transfer to scan.
+
+To process and scan each product individually, choose a specific product line. The :guilabel:`+#`
+button (in this case, :guilabel:`+10`) can be clicked to indicate receipt of that product, or the
+:guilabel:`pencil icon` can be clicked to open a new screen to edit that product line.
+
+From this screen, the product that's being received is listed. Under the product name, the
+:guilabel:`Quantity` line can be edited. Either change the `0` in the line to the desired quantity,
+or click the :guilabel:`/# Units` button (in this case, :guilabel:`/10 Units`) to automatically fill
+the quantity ordered from the :abbr:`PO (purchase order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-product-line-editor.png
+   :align: center
+   :alt: Product line editor for individual transfer in Barcode app.
```

Comment:
In this image, you're explaining a lot about the receipt, UI, different buttons, and I feel like we could benefit from more explanations about what's displayed in the image, by putting the image and related info in an example block. For example, we could explain more about the product that's being scanned.

Below is an example of what I was thinking! Feel free to ignore if you disagree (:

```suggestion
.. example::
   In the reception operation `WH/IN/00019`, `10 Units` of the `Barcode Product` is expected to be
   received. `[BARCODE_PROD]` is the :guilabel:`Internal Reference` set on the product form. Scan
   the barcode of the `Barcode Product` to receive one unit. Afterwards, click the pencil icon to
   manually enter the received quantities.

   .. image:: receipts_deliveries/receipts-deliveries-scanner-overview.png
      :align: center
      :alt: Overview of receipts in transfer to scan.
```